examples/app.py

```diff
@@ -62,7 +62,7 @@
         cd static
         npm install
         cd ..
-        npm install -g node-sass clean-css requirejs uglify-js
+        npm install -g node-sass@3.8.0 clean-css@3.4.24 requirejs uglify-js
```

Comment:
Hmm, judging from https://github.com/inveniosoftware/invenio/issues/3737#issuecomment-279947731 it's actually probably not fully sorted yet due to ``cleancss`` vs ``cleancssurl`` filters. So either we get the filters fixed in the package (e.g. search) or a new webassets package will have it fixed as well.